src/main/java/com/thealgorithms/devutils/nodes/SimpleTreeNode.java

```diff
@@ -57,12 +57,8 @@ public SimpleTreeNode(E data, SimpleTreeNode<E> parentNode) {
      * @param rightNode Value to which the nodes' right child reference will be
      * set.
      */
-    public SimpleTreeNode(
-        E data,
-        SimpleTreeNode<E> parentNode,
-        SimpleTreeNode<E> leftNode,
-        SimpleTreeNode<E> rightNode
-    ) {
+    public SimpleTreeNode(E data, SimpleTreeNode<E> parentNode, SimpleTreeNode<E> leftNode,
+        SimpleTreeNode<E> rightNode) {
```

Comment:
It's not formatted correctly, in my opinion. They should either all be on the same line, or all be on separate lines.
We really need to add an automatic linter in the build job to avoid breaking formatting in one place when fixing it in another.